packages/block-editor/src/components/link-control/index.js

```diff
@@ -112,10 +92,10 @@ function LinkControl( {
 		setInputValue( '' );
 	};
 
-	const handleDirectEntry = ( value ) => {
+	const handleDirectEntry = ( val ) => {
```

Comment:
I guess this was changed to avoid shadowing with the top-level prop? I'd typically not ever consider an abbreviation to be an improvement. I might have suggested something like `nextValue` if that's what it's intended to represent in this context.